### ISSUE TITLE
Write a JOSS article for opensource.com?

### ISSUE THREAD

**Issue by vmbrasseur** [openjournals/joss#357]
I'm an author and moderator for [opensource.com](https://opensource.com). The other day I read the [JOSS article](https://arxiv.org/pdf/1707.02264.pdf) reviewing the first year of the journal. It taught me a lot about what JOSS is all about. I heard of the journal when it launched but assumed it was just about research. I didn't realise it was repurposing the existing academic publication paradigms to enable citations for software. Quite clever.

Would someone on the JOSS team like to convert that academic piece into a shorter and more accessible for opensource.com? It would be great to let more people in free and open source software know about the existence of the journal and its work.

**Comment by danielskatz** [comment]
How short?  A couple of us have started working on an editorial article for a magazine...

**Comment by vmbrasseur** [comment]
We don't have hard word count limits. "Beginning to end" is my typical answer, but that's not helpful for you right now. Our editors would say "aim for 750" but all of my articles end up around 1500 words or so. Assume that if yours were to reach 2000 it might be too long and need to be split into 2 articles.

**Comment by danielskatz** [comment]
I'm happy to work on something for this - any other editors want to as well?

**Comment by karthik** [comment]
Sure, I'm down.

**Comment by Kevin-Mattheus-Moerman** [comment]
Sounds great. I can help too.

**Comment by arfon** [comment]
Count me in.

**Comment by danielskatz** [comment]
I've started https://docs.google.com/document/d/1QPaOJVePpUNRZJHL-pyzA7R9C3RBmSSumTjbvEsMguE/edit

**Comment by danielskatz** [comment]
I wrote most of a first draft now.

Again: https://docs.google.com/document/d/1QPaOJVePpUNRZJHL-pyzA7R9C3RBmSSumTjbvEsMguE/edit

Comments and help from others, such as @karthik @Kevin-Mattheus-Moerman @arfon would be welcome.

**Comment by danielskatz** [comment]
Also, @vmbrasseur - is this along the lines of what you were looking for, with the caveat that it's not complete and no one else from JOSS has yet weighed in?

**Comment by vmbrasseur** [comment]
@danielskatz So far it's looking pretty good. I do think that a problem statement will be needed close to the start of the article. Some in the audience may not be familiar with the publish-credit-tenure pipeline or that software isn't included in it (until JOSS).

**Comment by danielskatz** [comment]
There's another version there now, with comments addressed and changes accepted.

@karthik @Kevin-Mattheus-Moerman @arfon - can you take another look?

**Comment by arfon** [comment]
👍 this is looking great. Thanks @danielskatz!

**Comment by karthik** [comment]
LGTM

**Comment by danielskatz** [comment]
@Kevin-Mattheus-Moerman - please let us know if you still want to be involved in this.
If so, please check the draft (and add your email).

**Comment by cMadan** [comment]
I think this looks great! I've made a bunch of minor edits (with track changes).

**Comment by danielskatz** [comment]
@vmbrasseur - I think this is done.  What's the next step?

**Comment by labarba** [comment]
Great going, @danielskatz. If it's not too long already, maybe add at the end a short paragraph with the statistics of our successful first year? (and a link to the arXiv preprint?)

**Comment by vmbrasseur** [comment]
@danielskatz Next step: I get it to the opensource.com editors.

I'll need an email address which I can Cc on that.

Also, each author who would like to be listed on the article will need an opensource.com account. You can create one here: https://opensource.com/user/register

Once you have an account, please send me your username.

The site can only list one person as the primary author. To be honest, I don't know how many secondary authors the site can support (I'm not on that end of the operation). The editors will have more information about that.

So, in summary, I need:

* An email address which I can Cc on the message to the editors. This should be of the primary author.
* The opensource.com account name for each author

You can post those here or email me (my github username at opensource.com) if that works better for you.

**Comment by danielskatz** [comment]
The email to cc should be mine - d.katz@ieee.org
I've added my username to the draft, and have asked the other authors to as well.

**Comment by danielskatz** [comment]
In response to @labarba 's comment: 

> Great going, @danielskatz. If it's not too long already, maybe add at the end a short paragraph with the statistics of our successful first year? (and a link to the arXiv preprint?)

I've added a sentence at the end - comments welcome.

**Comment by danielskatz** [comment]
@vmbrasseur  - I think we are ready to go.

**Comment by vmbrasseur** [comment]
Email sent to the opensource.com editors, @danielskatz Cc'd.

You're all wonderful. Thank you for the article!

**Comment by vmbrasseur** [comment]
BTW, if any of the authors publishing in JOSS ever want to publish a non-academic piece about their software or research I'd be happy to help them do that at opensource.com. We love sharing stories about the amazing stuff people do with FOSS.

**Comment by danielskatz** [comment]
The article is now published - see https://opensource.com/article/17/12/journal-open-source-software

Thanks @vmbrasseur for the idea and help making this happen.
